examples/vertx-opentracing/README.md

```diff
@@ -12,58 +12,64 @@ Manager is performed via publishing a message on a shared destination.
 
 The other operation that can be performed by an end user is to list the current orders associated with an account id.
 
-== Run
+## Run
 
-First step is to start up the APM server. Then open a console window from this folder and run:
+First step is to start up the APM server. Then open a console window from this folder and either run the command directly using maven:
 
-[source,shell]
-----
+```shell
 $ export HAWKULAR_APM_URI=http://localhost:8080
 $ export HAWKULAR_APM_USERNAME=...
 $ export HAWKULAR_APM_PASSWORD=...
 $ mvn clean install
 $ mvn exec:java
-----
+```
 
-== Example requests
+or use docker:
+
+```shell
+$ docker build -t hawkular/apm-example-vertx-opentracing .
+$ docker run -it --rm -p 8180:8180 --add-host tracing-server:172.17.0.1 -e HAWKULAR_APM_URI=http://tracing-server:8080 -e HAWKULAR_APM_USERNAME=... -e HAWKULAR_APM_PASSWORD=... hawkular/apm-example-vertx-opentracing
```

Comment:
This do you want to add en entry to `/etc/hosts`? you can specify APM url directy with HAWKULAR_APM_URI so the `--add-host` is not necessary here.